src/States/GettingReadyState.js

```diff
@@ -130,6 +143,13 @@ function GettingReady() {
       friendDialogue.destroy();
     }
 
+    if (progress >= dialogueTree.length) {  //@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@ ISSUE!! @@@@@@@@@@@ -> progressing after loop
+      transitionToNextState();
+      return;
+    }
+
+    console.log(progress);
```

Comment:
We'll want to start being more judicious with the `console.log`s we end up committing into the code base. Make sure they're still helpful as we're going through development. Otherwise, be wary of keeping any in there since they'll also be run in its deployed state too.